Feature/gf 02

rbarusta opened the pull request:
se refactoriza metodo countingSort 💩

helttonrl committed:
Merge pull request #1 from swsapien/develop

no message

rbarusta committed:
se refactoriza countingSort

helttonrl commented on SortAlgoritms/SortAlgoritms/Algoritms/CountingSort.cs:
```diff
@@ -17,17 +16,59 @@ public class CountingSort : IAlgoritm
+            int[] values = new int[numberValues];
+            for (int i = 0; i < numberValues; i++)
+            {
+                values[i] = random.Next(numberValues); // Valores aleatorios dentro del rango
```
Debes incluir valores negativos también. El rango es del -100 al 100.

rbarusta replied:
done ✓

helttonrl reviewed (changes requested):
Deben cambiar el código de acuerdo con los comentarios realizados.

rbarusta committed:
GF-02
se refactoriza metodo para obtener numeros random, contemplando tambien numeros negaticos hasta el -100
